### ISSUE TITLE
Input Parameters: Role and Name properties are not defined for 'Add Pram(+)' and 'Close Pram(x)' Icon controls present under  'Input Parameters' blade.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1049]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1242519

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section and select any of the item then some new menu item appears in the top like New SQL Query, New Stored Procedure.
Navigate to "New Stored Procedure" menu item and select it.
Navigate to Stored Procedure ID edit field and give input and select Save menu item.
Now Navigate to "Execute" control and select it.
Navigate through all the controls under "Input Parameters" blade.
Navigate to Add Pram(+) and Close Pram(x) icon controls.
Press F12 key to open Developers tool and inspect the code.
Verify the issue.

Actual Result:
Role and Name properties are not defined for 'Add Pram(+)' and 'Close Pram(x)' Icon controls present under  'Input Parameters' blade.

Expected Result:
Role and Name properties should be defined for 'Add Pram(+)' and 'Close Pram(x)' icon controls present under 'Input Parameters' blade. Name='Add Parameter' and role='button' should be defined for 'Add Pram(+)' icon control. Same for  'Close Pram(x)' icon controls.

![image](https://user-images.githubusercontent.com/8519407/132716487-a32763c8-3540-4b13-bbc5-47ae44442d96.png)
